modules/ROOT/pages/functions/dev_guide/quarkus/invocation-examples-quarkus.adoc

```diff
@@ -0,0 +1,120 @@
+// Module included in the following assemblies
+//
+// /functions/dev_guide/develop-quarkus.adoc
+
+// [id="invocation-examples-quarkus_{context}"]
+= Invocation examples
+
+This code defines the `withBeans`, `withCloudEvent`, and `withBinary` functions:
+
+.Example
+[source,java]
+----
+import io.quarkus.funqy.Funq;
+import io.quarkus.funqy.knative.events.CloudEvent;
+
+public class Input {
+    private String message;
+
+    // getters and setters
+}
+
+public class Output {
+    private String message;
+
+    // getters and setters
+}
+
+public class Functions {
+    @Funq
+    public Output withBeans(Input in) {
+        // function body
+    }
+
+    @Funq
+    public CloudEvent<Output> withCloudEvent(CloudEvent<Input> in) {
+        // function body
+    }
+
+    @Funq
+    void withBinary(byte[] in) {
+        // function body
+    }
+}
+----
+
+* The `withBeans` function of the `Functions` class can be invoked by:
+
+** An HTTP POST request with JSON body:
++
+[source]
+----
+$ curl "http://localhost:8080/" -X POST \
+    -H "Content-Type: application/json" \
+    -d '{"message": "Hello there."}'
+----
+
+** An HTTP GET request with query parameters:
++
+[source]
+----
+$ curl "http://localhost:8080?message=Hello%20there." -X GET
+----
+
+** A `CloudEvent` in binary encoding:
++
+[source]
+----
+curl "http://localhost:8080/" -X POST \
+  -H "Content-Type: application/json" \
+  -H "Ce-SpecVersion: 1.0" \
+  -H "Ce-Type: my-type" \
+  -H "Ce-Source: cURL" \
+  -H "Ce-Id: 42" \
+  -d '{"message": "Hello there."}'
+----
+
+** A `CloudEvent` in JSON encoding:
++
+[source]
+----
+$ curl http://localhost:8080/ \
+    -H "Content-Type: application/cloudevents+json" \
+    -d '{ "data": {"message":"Hello there."},
+          "datacontenttype": "application/json",
+          "id": "42",
+          "source": "curl",
+          "type": "my-type",
+          "specversion": "1.0"}'
+----
+
+* The `withCloudEvent` function of the `Functions` class can be invoked with `CloudEvent` similarly to the `withBeans` function.
```

Comment:
There is a small difference though. The `withCloudEvent` function cannot be invoked by plain (non-CE) HTTP. Only the last two examples for `withBeans` are valid here. The first two wouldn't work.

Reply:
Hmm, I thought specifying "with `CloudEvent`" was enough. However, you are right, it's better to be explicit, so how about:

* The `withCloudEvent` function of the `Functions` class can be invoked with `CloudEvent` similarly to the `withBeans` function. However, unlike `withBeans`, `withCloudEvent` cannot be invoked with a plain HTTP request.

Reply:
I think that suggested change is good.

Reply:
Thanks Matěj, applied the suggestion in https://github.com/openshift-knative/docs/pull/62